slu/slu/dev/train.py

```diff
@@ -180,11 +184,19 @@ def train_intent_classifier(args: argparse.Namespace) -> None:
             """.strip()
         )
 
-    workflow = SLUPipeline(config).get_workflow(purpose=const.TRAIN, epochs=epochs)
+    workflow = SLUPipeline(config).get_workflow(
+        purpose=const.TRAIN, epochs=epochs)
 
     logger.info("Preparing dataset.")
-    dataset = dataset or config.get_dataset(const.CLASSIFICATION, f"{const.TRAIN}.csv")
+    dataset = dataset or config.get_dataset(
+        const.CLASSIFICATION, f"{const.TRAIN}.csv")
     data_frame = pd.read_csv(dataset)
+    data_frame = data_frame[~data_frame[const.TAG].isin(
+        config.tasks.classification.skip)]
+    data_frame = data_frame.replace(
+        {const.TAG: config.tasks.classification.alias})
+    logger.info(
+        f"Model will be trained for the following classes:\n{data_frame[const.TAG].value_counts(dropna=False)}")
```

Comment:
we expect all intent related modification to happen inside [`create_data_splits`](https://github.com/skit-ai/dialogy-template-simple-transformers/blob/259b2c12953da08db87444f4ee5d31da1f7115a5/slu/slu/dev/train.py#L90) function only, and skip intents one is already happening there so just addition for alias after skip would be great.